posturebest/Views/Torso3DView.swift

```diff
@@ -30,22 +33,11 @@ struct Model3DView: UIViewRepresentable {
     }
     
     func printNodeHierarchy(node: SCNNode, depth: Int = 0) {
-        let indentation = String(repeating: "  ", count: depth)
-        
-        // Print node details
-        print("\(indentation)Node: \(node.name ?? "Unnamed Node")")
-        print("\(indentation)  Position: \(node.position)")
-        print("\(indentation)  Rotation: \(node.rotation)")
-        print("\(indentation)  Scale: \(node.scale)")
-        
-        // Recursively print child nodes
-         for childNode in node.childNodes {
-            printNodeHierarchy(node: childNode, depth: depth + 1)
-        }
+        _ = String(repeating: "  ", count: depth)
```

Comment:
is this whole function needed anymore? to my understanding, this isn't printing anything or generating anything needed anymore if we get rid of the print statements